error: diamond operator is not supported in -source 1.6

When creating a new JME project (or JmeTests) I can see that nbproject/project.properties contains:
javac.source=1.6
javac.target=1.6

![image](https://cloud.githubusercontent.com/assets/1143272/18685549/d91f7ba6-7f78-11e6-90bd-2c473d71b223.png)


This default setting gives errors like:

warning: [options] bootstrap class path not set in conjunction with -source 1.6
/home/j/3D-briller/jme_proj_3.1-beta1-b002-

SNAPSHOT/JmeTests/src/jme3test/app/TestCloneSpatial.java:176: error: diamond operator is not supported in -source 1.6

So, please change all the new project templates's values to
javac.source=1.8
javac.target=1.8

--- a/BasicGameTemplate/nbproject/project.properties
+++ b/BasicGameTemplate/nbproject/project.properties
@@ -49,8 +49,8 @@ javac.compilerargs=
 javac.deprecation=false
 javac.processorpath=\
     ${javac.classpath}
-javac.source=1.6
-javac.target=1.6
+javac.source=1.7
+javac.target=1.7
 javac.test.classpath=\
     ${javac.classpath}:\
     ${build.classes.dir}

